src/webgl/loading.js

```diff
@@ -103,6 +103,8 @@ import './p5.Geometry';
  * @param  {String} [fileType]
  * @return {p5.Geometry} the <a href="#/p5.Geometry">p5.Geometry</a> object
  */
+
+let materials = {}; //currently local
```

Comment:
Do we use this btw? Looks like we always pass in materials as a parameter

Reply:
Ah, yes I had to remove it , missed it though. I'll remove it.